Obsolete example (#6)
DevExpress-Examples/obsolete-blazor-create-datagrid-at-runtime: Readme.md

The Data Grid was moved to maintenance support mode. No new features/capabilities will be added to this component. We recommend that you [migrate](https://docs.devexpress.com/Blazor/403162/grid/migrate-from-data-grid-to-grid) to the [Grid](https://docs.devexpress.com/Blazor/403143/grid) component. 

> We do not recommend to generate columns at runtime as Blazor cannot guarantee fixed column order. Manage the columns [Visible](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridColumn.Visible) properties instead.

Reviewer: ```suggestion
We do not recommend that you generate columns at runtime as Blazor cannot guarantee fixed column order. Manage the columns [Visible](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridColumn.Visible) properties instead.

```

I'm not sure about the `Manage the columns [Visible] properties instead` part - Is this 'columns' Visible properties'? And how should you manage them?

Author: The idea is to create all required columns in markup and use the Visible property of columns to show\hide columns as one wishes.

Reviewer: Ok, but how is this connected to generating columns at runtime?

Author: Some users generate columns at runtime when they want to show additional columns after the component is created. We do not recommend this approach because code that relies on column order can face unexpected issues as Blazor does not keep the column order strictly. It is better to create all columns in advance and set Visible=true when needed.

```suggestion
We do not recommend that you generate columns at runtime as Blazor cannot guarantee fixed column order. Use the column [Visible](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxGridColumn.Visible) properties instead.

```

Reviewer: Ok
